src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/changes-since-v3.mdx

```diff
@@ -21,118 +17,6 @@ Data reported by the Kubernetes Integration version 3 hasn't changed since versi
-
-Scraping configuration for the etcd CP component looks like the following where the same structure and features applies for all components:
-
-```yaml
-config:
-  etcd:
-    enabled: true
-    autodiscover:
-      - selector: "tier=control-plane,component=etcd"
-        namespace: kube-system
-        matchNode: true
-        endpoints:
-          - url: https://localhost:4001
-            insecureSkipVerify: true
-            auth:
-              type: bearer
-          - url: http://localhost:2381
-    staticEndpoint:
-      url: https://url:port
-      insecureSkipVerify: true
-      auth: {}
-```
-
-If `staticEndpoint` is set, the component will try to scrape it. If it can't hit the endpoint, the integration will fail so there are no silent errors when manual endpoints are configured.
-
-If `staticEndpoint` is not set, the component will iterate over the autodiscover entries looking for the first pod that matches the `selector` in the specified `namespace`, and optionally is running in the same node of the DaemonSet (if `matchNode` is set to `true`). After a pod is discovered, the component probes, issuing an http `HEAD` request, the listed endpoints in order and scrapes the first successful probed one using the authorization type selected.
-
-While above we show a config excerpt for the `etcd` component, the scraping logic is the same for other components.
-
-For more detailed instructions on how to configure control plane monitoring, please check the [control plane monitoring](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring) page.
-
-### Helm Charts [#helm-charts]
-
-Helm is the primary means we offer to deploy our solution into your clusters. Chart complexity was also significantly increased from the previous version, where it only had to manage one DaemonSet. Now, it has to manage one deployment and two DaemonSets where each has slightly different configurations. This will give you more flexibilty to adapt the solution to your needs, without the need to apply manual patches on top of the chart and the generated manifests.
-
-Some of the new features that our new Helm chart exposes are:
-
-* Full control of the `securityContext` for all pods
-* Full control of pod `labels` and `annotations` for all pods
-* Ability to add extra environment variables, `volumes`, and `volumeMounts`
-* Full control on the integration configuration, including which endpoints are reached, autodiscovery behavior, and scraping intervals
-* Better alignment with Helm idioms and standards
-
-You can check full details on all the switches that can be flipped in the [Chart's `README.md`](https://github.com/newrelic/nri-kubernetes/blob/main/charts/newrelic-infrastructure/README.md).
-
```

Comment:
I didn't see this information moved elsewhere, what is the reasoning for removing this section above?